mongodb/map.jinja

```diff
@@ -1,14 +1,25 @@
 {% set mongodb = salt['grains.filter_by']({
     'default': {
-        'pkg': 'mongodb',
-        'service': 'mongodb',
-        'conf_file': '/etc/mongodb/mongodb.conf',
-        'config': {
-            'key': 'value',
-        },
+      'key': '7F0CEB10',
+      'keyserver': 'keyserver.ubuntu.com',
```

Comment:
Is this the same keyserver that would be used for RHEL distros? Also, if the RHEL distro needs the GPG key then that has to be handled as a separate step. See https://github.com/mitodl/elasticsearch-formula/blob/master/elasticsearch/repository.sls for reference.